frontend/components/Contribute.vue

```diff
@@ -120,6 +154,25 @@ export default {
         }
       })
     },
+    handlePlaceClick(e, data, type) {
+      this.hideModal()
+      this.$store.commit('contribute/setIsDrawMode', true)
+      this.$router.push({
+        path: '/contribute',
+        query: {
+          mode: data,
+          type
+        }
+      })
+    },
+    validateArtist($event) {
+      this.hideModal()
+      if (this.validatedArtist) {
```

Comment:
This if block is still empty. It might be a good idea to indicate if this is a `Work in Progress`?